presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -303,10 +318,12 @@ public SslTrustStorePath()
     {
         private static final Predicate<Properties> IF_TRUST_STORE =
                 checkedPredicate(properties -> SSL_TRUST_STORE_PATH.getValue(properties).isPresent());
+        private static final Predicate<Properties> IF_SSL_VERIFICATION_ENABLED =
+                checkedPredicate(properties -> !SSL_VERIFICATION.getValue(properties).orElse(SslVerificationMode.FULL).equals(SslVerificationMode.NONE));
```

Comment:
you should use default value when `SSL_VERIFICATION` is not set explicitly. Also this predicate should be defined `SslVerificationMode` and reused instead of being copied 3 times.